hl13/code/modules/projectiles/projectile/pistol/pistol.dm

```diff
@@ -1,6 +1,13 @@
 /obj/projectile/bullet/hl13/hl9mm
 	name = "9mm bullet"
-	damage = 30
+	damage = 5
```

Comment:
weapons were supposed to pack a punch since we were going for a realistic kind of gameplay were a gunshot would really hurt you without protection.. Keep in mind that 5 is basically a sting and its really low.

Reply:
tweaked back to 30